src/get-registration.js

```diff
@@ -1,5 +1,17 @@
 import { isObject } from './utils'
-import { ALIASES } from './constants'
+
+export const STATUSES = {
+  inactive  : 'inactive',
+  open      : 'open',
+  update    : 'loading',
+  loading   : 'update',
+  done      : 'resolve',
+  resolve   : 'done',
+  reject    : 'error',
+  error     : 'reject',
+  cancel    : 'cancelled',
+  cancelled : 'cancel'
```

Comment:
Ah. Why did I do this.  This should still be called aliases.

This is so that if you subscribe to `reject`, or `error`, they hit the same handler.